### PR TITLE
Remove restaurant list and update UI

### PR DESCRIPTION
**Pull request by jieliangang**
- Remove restaurant list
- Randomise
- Move most information to restaurant section
- Arrows to navigate

Still in progress. Do provide feedback and any recommendations for the UI.

As usual, deployed at https://jang-sps-summer20.appspot.com/

### CODE REVIEW

**Comment by daekoon** [comment]
Looks great so far!
One suggestion I have is that for the ratings, we may want to show it as a fraction (4.5/5) so that people know what the max score is. When the icons are used in Google Maps the max score is shown implicitly, but here since we are using numbers we should display the max score as well.

**Comment by daekoon** [comment]
Is this still in progress?

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -38,17 +38,8 @@
...
-          <span class="iconify" data-icon="bx:bxs-direction-right" data-inline="false" data-width="38px" data-height="38px" style="color: blue;"></span>
-        </a>
-        <a id="whatsapp-share" onclick="gtag('event', 'share', {'method': 'Whatsapp'})">
-          <span class="iconify" data-icon="logos:whatsapp" data-inline="false" data-width="40px" data-height="40px"></span>
-        </a>
-        <a id="telegram-share" onclick="gtag('event', 'share', {'method': 'Telegram'})">
-          <span class="iconify" data-icon="logos:telegram" data-inline="false" data-width="35px" data-height="35px"></span>
-        </a>
+      <div id="infowindow-restaurant-content">
+        <div id="info-restaurant-name">${name}</div>
```
Comment (lolfuljames) [inline src/main/webapp/index.html]: We need to add in the "hide" class, and remove the extra text

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -141,54 +142,115 @@ h1 {
...
+  width: 1rem;
+  height: 1rem;
 }
 
-#restaurant-list .selected:hover {
-  background: #FFB74D;
+.rating-active {
+  display: block;
+  width: 1rem;
+  height: 1rem;
```
Comment (lolfuljames) [inline src/main/webapp/style.css:219]: We could use a generic ratings class for this, and use the classes 'show' and 'hide' to perform the hiding/showing, but this is fine too.

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -141,54 +142,115 @@ h1 {
...
-  background-color: #E3F2FD;
-  margin: 10px 0;
-  border-radius: 8px;
-  cursor: pointer;
+#restaurant-image{
+  height: 200px;
+  width: calc(25vw - 80px);
+  object-fit: cover;
+  margin: .5rem 0;
+}
```
Comment (lolfuljames) [inline src/main/webapp/style.css]: It would be great if we could add rounded corners using border-radius for this, it'll look cleaner

**Review by lolfuljames — COMMENTED** [review]
Just some comments

File: src/main/webapp/style.css [hunk 4]
```diff
@@ -341,17 +342,24 @@ h1 {
...
     max-height: 40px;
     overflow: hidden;
   }
 
-  #result-container.result-hidden #result-hide-button {
-    margin-bottom: 10px;
-  }
+  /*#result-container.result-hidden #result-hide-button {*/
+  /*  margin-bottom: 20px;*/
+  /*}*/
```
Comment (lolfuljames) [inline src/main/webapp/style.css]: is this still needed?

**Review by lolfuljames — APPROVED** [review]
Can merge after you've resolved some of the minor comments 👍

File: src/main/webapp/script.js [hunk 5]
```diff
@@ -117,102 +115,171 @@ function initMap() {
...
+}
+
+function prevRestaurant() {
+  restaurantIndex = (restaurantIndex === 0) ? restaurants.length - 1 : restaurantIndex - 1;
+  updateRestaurantDetails(restaurantIndex);
+}
+
+function setRestaurantTab(hidden) {
+  const resultContainerElement = document.getElementById("result-container");
+  if(hidden) {
```
Comment (daekoon) [inline src/main/webapp/script.js:238]: Missing space before the bracket (

Comment (daekoon) [reply src/main/webapp/script.js:238]: Actually ignore this, half the code has space before bracket and half don't. Let's fix the linting in a separate PR

**Review by daekoon — APPROVED** [review]
LGTM!

**Pull request merged**